File: exercises/dani/q1a.cpp
#include <iostream>
using namespace std;


int main() {
    double Cost, Revenue, Amount, Profit, Loss;
    cout << "Enter total cost: ";
    cin >> Cost;
    cout << "Enter total revenue: ";
    cin >> Revenue;
    Amount = Revenue - Cost;

    if(Amount > 0) {
       Profit = Amount;
       cout << "The profit is GHC " << Profit << endl;
    } else {
       Loss = abs(Amount);
       cout << "The loss is GHC " << Loss << endl;
    }
}

File: exercises/dani/q1a.py
Cost = float(input("Enter total cost: "))
Revenue = float(input("Enter total revenue: "))
Amount = Revenue - Cost
if Amount > 0:
    Profit = Amount
    print("The profit is GHC " + str(Profit))
else:
    Loss = abs(Amount)
    print("The loss is GHC " + str(Loss))
